content/en/docs/getting-started/running.md

```diff
@@ -160,6 +160,44 @@ docker run --rm -i -t \
     falcosecurity/falco:latest
 ```
 
+It is also possible to use `falco-no-driver` and `falco-driver-loader` images in fully privileged mode.
+This may be desirable in environments which do not allow the full falco image due to space, resource, security or policy constraints.
+You can load the eBPF probe or kernel module in its own temporary container as below:
+
+```shell
+docker pull falcosecurity/falco-driver-loader:latest
+docker run --rm -i -t \
+    --privileged \
+    # Uncomment the below to use the eBPF probe or delete it if using the kernel module
+    #-e FALCO_BPF_PROBE="" \
```

Comment:
```suggestion
    #-e FALCO_BPF_PROBE="" -v /root/.falco:/root/.falco \
```

The behavior is slightly different when it comes to the bpf probe. We have to share the path where the probe will be stored.

Reply:
My fault, you are right I should have caught that. Thanks!